twitter-export-image-fill.py

```diff
@@ -58,6 +58,24 @@
 
 image_count_global = 0
 
+# Function to download an avatar (from tweet or retweet)
+
+def download_avatar(user):
+  avatar_url = user['profile_image_url_https']
+  extension = os.path.splitext(avatar_url)[1]
+  screen_name = user['screen_name']
+  local_filename = "img/avatars/%s%s" % (screen_name, extension)
+  downloaded = 0
```

Comment:
I was having it return 0 or 1 to indicate how many avatars were downloaded without additional logic. But now that avatar counts are no longer needed I can have it return True/False depending on whether it updated the user stanza or not.